Topics/math-fundamentals.md

```diff
@@ -0,0 +1,172 @@
+
+## Prerequisites
+The previous Paradiso guides.
+
+## Time to complete
+10 minutes.
+
+# Index
+- [Motivation](#Motivation)
+- [Courses](#Courses)
+  - [Calculus](#Calculus)
+  - [Linear Algebra and Matrix Algebra](#Linear-Algebra)
+  - [Optimization](#Optimization)
+- [Ask Questions](#Ask-Questions)
+- [Conclusions](#Conclusions)
+
+Let's dive right in!
+ 
+### Motivation
+
+Before anything else, watch [this](https://www.youtube.com/watch?v=VIbjHIGMjQM
+) and [this](https://www.youtube.com/watch?v=8CX-Q0gtSp8) awesome Ted Talks :-)
+
+**The entire field of Data Science is based on mathematics and statistics.**
+
+This phrase shouldn't frighten you, though, because as you'll find out, you probably have much of the knowledge you need to start experimenting!
+
+**In fact:** 
+
+If you've done a second-degree school (you know matrices and derivatives, for example), you probably just need to refresh yourself on the things you already know and go deeper into some things you may not have seen at school. 
+
+In general, if you have a scientific degree (engineering, computer science, physics, etc.), it will probably be just a review, as it will be the Calculus, Linear Algebra, and Matrix Algebra courses are suggested within this guide.
+
+Whatever your starting level, however, it is still recommended to go through all the resources organized here, _to make sure you do not have "fundamental knowledge holes"._
+
+Remember to read and try to implement some ideas of the [**Virgilio's Learning Strategy - Learning to Learn**](teaching.md)
+
+## Courses
+### Calculus
+
+The first macro-topic you need to be confident with is [Calculus](https://en.wikipedia.org/wiki/Calculus).
+
+It would be impossible to list how many Machine Learning system properties are based on its rules, like many of the scientific fields!
+
+The process of training an ML model it's nothing more than Calculus rules applied to statistical models! 
+
+**Lucky you!** 
+You can find great free courses offered by the professors of the [MIT](http://www.mit.edu/)(Massachusetts Institute of Technology), through the index page [MIT OpenCourseWare](https://ocw.mit.edu/index.htm).
+
+The course we suggest in particular is the following:
+
+[**Single Variable Calculus**](https://ocw.mit.edu/courses/mathematics/18-01sc-single-variable-calculus-fall-2010/)
+
+There are both the video lessons and the notes of the course.
+
+Once you've taken the full course, watch this awesome playlist from [3Blue1Brown](https://www.youtube.com/channel/UCYO_jab_esuFRV4b17AJtAw):
+
+[**The Essence of Calculus**](https://www.youtube.com/watch?v=WUvTyaaNkzM&list=PLZHQObOWTQDMsr9K-rj53DwVRMYO3t5Yr)
+
+Thanks to these two resources, you should be able to understand most of the mathematical reasons that underpin today's Machine Learning systems and build them to develop powerful Data Science projects
+
+If you want to dive deeper, here you can find a [Multivariable Calculus]() course with notes and videos, again from MIT:
```

Comment:
Link missing, which this still work?: [Multivariable Calculus]()

Reply:
Oh thanks. It's the definition from Wikipedia.